package.json

```diff
@@ -1,18 +1,17 @@
 {
-  "name": "taco-simulate-server",
-  "version": "0.1.3",
-  "description": "Server and app host module for taco-simulate (Cordova plugin simulation).",
-  "main": "src/server.js",
+  "name": "cordova-simulate",
+  "version": "0.2.0",
```

Comment:
Sounds good. Going forward, this will mean each time we do a release we remove the `-dev` (and either going with this version or bump it if the changes warrant that), then immediately following the release we bump the patch version and add the `-dev` again.